torch/_dynamo/variables/functions.py

```diff
@@ -1039,6 +1049,52 @@ def call_grid(self, grid, meta, tx):
         grid = grid.call_function(tx, [meta], {})
         return grid
 
+    # We use this function to wrap call_prune_configs
+    def call_user_defined_fn(self, user_fn, args, kwargs, tx, variable):
+        from .builder import VariableBuilder
+
+        wrapped_user_function = VariableBuilder(tx, variable.source)._wrap(user_fn)
```

Comment:
Don't use VariableBuilder here -- it looks like the only usage is with `user_fn=do_prune_configs`. The source of this is not variable.source (where variable is the TritonKernelVariable).

I don't think do_prune_configs doesn't need a source. Is it possible to use SourcelessBuilder on this? If not then it is OK to directly create a UserDefinedFunctionVariable